docs/zh/quickstart/sdk/java_sdk.md

```diff
@@ -1,5 +1,7 @@
 # Java SDK
 
+Java SDK默认执行模式为在线。

```

Comment:
the default execute_mode in java sdk is online?

Reply:
https://github.com/4paradigm/OpenMLDB/blob/68e1f7babd2cbd7ba81c872a31772524193dbe1f/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/jdbc/SQLConnection.java#L45-L52 jdbc will be online, I'll make it more accurately